sentences/ro/_common.yaml

```
@@ -58,6 +60,7 @@ expansion_rules:
   temperature: "{temperature} [[de] grad[e]] [{temperature_unit}]"
   name: "({name}[(ul|a)])"
   maximum: "(maxim[(ă|a|um)] [posibil[(ă|a)]])"
+  jumătate: "(jumătate | jumatate | jumate)"
```

keys should only contain english alphabet characters. Also, please contract diacritics
```suggestion
  jumatate: "(jum(ă|a)tate | jumate)"
```